package.json

```diff
@@ -37,7 +37,6 @@
     ]
   },
   "scripts": {
-    "prepare": "npm run compile",
```

Comment:
🤔 `prepare` is a hook that runs after `npm install` -- is there a chance that this breaks other pipelines/ docs that assume an install will also build assets etc.?

No particular issue with this being removed if not

Reply:
Yeah, as I said in https://github.com/alphagov/pay-frontend/pull/2715, I don't think this is a useful thing to do. I'll copy-pasta that reasoning here. Github actions runs that task separately (otherwise tests would fail).